src/api/app/models/project.rb

```diff
@@ -1395,6 +1395,7 @@ def image_template?
   end
 
   def signing_key(type:)
+    return nil if type.nil?
```

Comment:
```suggestion
    return nil if type.nil?
    
```
I think you need a newline to make the linter happy